.gitignore

```diff
@@ -67,3 +67,4 @@ jsdist/
 cypress/videos/
 cypress/screenshots/
 slides
+sockpuppet/static/
```

Comment:
This can't be ignored, as these need to be committed.

Reply:
ok!